rack-protection/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+## 2.0.0 / 2017-04-08
```

Comment:
I'm not sure the point of having 3 separate changelogs either, can we just use `sinatra/CHANGELOG.md`?